src/Platform/Microsoft.Testing.Platform/Configurations/AggregatedConfiguration.cs

```diff
@@ -64,7 +70,11 @@ public void SetTestHostWorkingDirectory(string workingDirectory)
         _testHostWorkingDirectory = workingDirectory;
     }
 
-    public async Task CheckTestResultsDirectoryOverrideAndCreateItAsync(ICommandLineOptions commandLineOptions, IRuntime runtime, ServiceProvider serviceProvider)
+    public async Task CheckTestResultsDirectoryOverrideAndCreateItAsync(
+        ICommandLineOptions commandLineOptions,
+        IRuntime runtime,
+        IFileSystem fileSystem,
```

Comment:
Just double checked and file system and runtime are both system API wrappers and so are available from the start. I would prefer to have them moved to the ctor.

Reply:
Closing this as you already did a PR for this, thanks !